lib/perl/Genome/Model/Tools/Manta/Base.pm

```diff
@@ -0,0 +1,246 @@
+package Genome::Model::Tools::Manta::Base;
+
+use strict;
+use warnings;
+
+use Genome;
+use Carp qw(confess);
+
+class Genome::Model::Tools::Manta::Base {
+    is => 'Command',
+    is_abstract => 1,
+    attributes_have => [
+        tool_arg_name => {
+            is => 'String',
+            is_optional => 1,
+        },
+        tool_input_file => {
+            is => 'Boolean',
+            is_optional => 1,
+        },
+    ],
+    has_param => [
+        version => {
+            is => 'Version',
+            doc => 'The version of Manta to use.',
+            valid_values => ['0.29.6'],
```

Comment:
It might be nice if this could somehow rely on `&available_versions` (or `sort keys %TOOL_VERSIONS` directly) instead of repeating the values.